apps/web/pages/org/[id]/[pk]/event.tsx

```diff
@@ -0,0 +1,42 @@
+import { Editor } from 'novel';
+import { DashboardLayout } from '@app/layout';
+import { ENV } from '@app/config';
+import { Button } from '@app/components/ui/Button';
+
+const defaultEditorContent = {
+    type: 'doc',
+    content: [
+        {
+            type: 'heading',
+            attrs: { level: 2 },
+            content: [{ type: 'text', text: 'Add Your Event Info Here' }],
+        },
+    ],
+};
+
+const Event = () => {
+    const onPublish = () => {
+        // novel store the data in local storage
+        const data = localStorage.getItem('novel__content');
+    };
```

Comment:
local storage return string to parse it and add API to send the data to

Reply:
also since each event have diffrent novel local storage they will save that data into same key , we will need to send the data to backend on each interval and remove the local storage or else it will be hard to manage  the novel for each event